README.md

```diff
@@ -94,12 +94,15 @@ and health-checking mechanisms.
 * Build Motr RPMs
 
   * From sources
+    **NOTE:** Installing motr RPMs other than "cortx-motr-test*.rpm"
     ```sh
     git clone --recursive https://github.com/Seagate/cortx-motr.git
     cd cortx-motr
 
     scripts/m0 make rpms
-    sudo rpm -ivh ~/rpmbuild/RPMS/x86_64/cortx-motr-*.rpm
+    cd ~/rpmbuild/RPMS/x86_64/
+    sudo rpm -ivh $(ls | grep  "cortx-motr-" | grep -v "cortx-motr-tests")
```

Comment:
Before this we used to  install both motr and motr-test rpms?
Or motr did not use to create the test rpms, and now it has started creating this additional test rpm, so the installation is failing?
if we dont install the motr-test rpm's, then if someone wants to run motr-tests which are probably provided by this motr-test rpm, then what will happen? will it still run or fail?
how motr is is doing it, in their installations?

Reply:
@suvratjoshi I think this is not required when testing HARE. This README's scope is hare. This has been confirmed with @atulsdeshmukh2312 . I think installation of motr-test rpm is in the scope of Motr README. For your questions on whether its gettign generated recently - @madhavemuri  can help here.